ui/frontend/Playground.tsx

```diff
@@ -88,15 +90,43 @@ const ResizableArea: React.FC = () => {
 };
 
 const Playground: React.FC = () => {
-  const showNotifications = useSelector(selectors.anyNotificationsToShowSelector);
+  const showNotifications = useSelector(
+    selectors.anyNotificationsToShowSelector
+  );
+
+  const dispatch = useAppDispatch();
+  const handleRustFmt = useCallback((_event) => {
+    dispatch(actions.performFormat());
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+  const handleClippy = useCallback((_event) => {
+    dispatch(actions.performClippy());
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+  const handleMiri = useCallback((_event) => {
+    dispatch(actions.performMiri());
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+  const handleMacroExpansion = useCallback((_event) => {
+    dispatch(actions.performMacroExpansion());
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  const shortcutMap = new Map([
+    [['Control', 'Alt', 'f'], handleRustFmt],
```

Comment:
FWIW, `shift+alt+f` is the default VSCode shortcut for formatting a document on macOS and Windows (but not Linux, it seems?), so users of the Monaco editor might find that more familiar, although it doesn't use `Control` like the shortcuts propsed here.

On macOS at least (and I would guess on other OSes as well), some of these shortcuts seem to have uses already within the editor:
* `cmd+alt+f` seems to open a find+replace dialog in `ace` and `monaco`
* `cmd+alt+c` toggles case-sensitivity for searches in `monaco`

----

This PR looks like it has been open a while without much movement, but I just wanted to provide some feedback on the choice of shortcuts. If there is any interest in reviving this I think it would definitely be a nice feature to have!